konfera/event/views.py

```diff
@@ -74,15 +88,50 @@ def dispatch(self, *args, **kwargs):
 
     def post(self, *args, **kwargs):
         context = self.get_context_data(**kwargs)
+        notify = getattr(django_settings, 'PROPOSAL_NOTIFY', getattr(settings, 'PROPOSAL_NOTIFY'))
+        template = EmailTemplate.objects.get(name='confirm_proposal')
 
         if context['speaker_form'].is_valid() and context['talk_form'].is_valid():
-            speaker_instance = context['speaker_form'].save()
+            speaker = context['speaker_form'].save()
             talk_instance = context['talk_form'].save(commit=False)
-            talk_instance.primary_speaker = speaker_instance
+            talk_instance.primary_speaker = speaker
             talk_instance.event = context['event']
             talk_instance.status = talk_instance.status or Talk.CFP
             talk_instance.save()
-            messages.success(self.request, self.message_text)
+
+            if notify:
+                event_url = self.request.build_absolute_uri(reverse('event_details', args=[self.event.slug]))
+                edit_url = self.request.build_absolute_uri(reverse('event_cfp_edit_form',
+                                                                   args=[self.event.slug, self.event.uuid]))
+
+                subject = _('Proposal for {event} has been submitted'.format(event=self.event.title))
+                template_data = {'first_name': speaker.first_name,
+                                 'last_name': speaker.last_name,
+                                 'event': self.event.title,
+                                 'talk': talk_instance.title,
+                                 'event_url': event_url,
+                                 'edit_url': edit_url,
+                                 'end_call': self.event.cfp_end}
+                text_content = template.text_template.format(**template_data)
+                html_content = template.html_template.format(**template_data)
+
+                msg = EmailMultiAlternatives(subject, text_content, to=[speaker.email],
+                                             bcc=settings.REGISTER_EMAIL_BCC)
```

Comment:
I thought BCC can be universal for all notifications, but you are right - I will make a new bcc for this  notifications